MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -2918,8 +2929,20 @@ public virtual long GetRunTimeTicksForPlayState()
             return RunTimeTicks ?? 0;
         }
 
-        // Possible types of extra videos
-        public static readonly IReadOnlyCollection<ExtraType> DisplayExtraTypes = new[] { Model.Entities.ExtraType.BehindTheScenes, Model.Entities.ExtraType.Clip, Model.Entities.ExtraType.DeletedScene, Model.Entities.ExtraType.Interview, Model.Entities.ExtraType.Sample, Model.Entities.ExtraType.Scene };
+        /// <summary>
+        /// Extra types that should be counted and displayed as "Special Features" in the UI.
+        /// </summary>
+        public static readonly IReadOnlyCollection<ExtraType?> DisplayExtraTypes = new HashSet<ExtraType?>
+        {
+            null,
+            0,
```

Comment:
I don't think including `0` here should be required since I fixed the issue with the `ExtraType` being set to `0`, but I left it here anyways for future-proofing

Reply:
Seems like we should just include a generic type rather than a null value.

Reply:
Finally figured this out. I've added a new `ExtraType.Unknown` and set that value where appropriate. Also updated the `ExtraResolver` to try parsing the extra type based on the containing directory name